faq/mpi-apps.inc

```diff
@@ -1007,3 +1007,16 @@ to some of the available packages:
 over time; projects come, and projects go.  Your best bet these days
 is simply to use Google to find MPI bindings and application-level
 programming interfaces.";
+
+/////////////////////////////////////////////////////////////////////////
+
+$q[] = "Why does my legacy MPI application fail to compile with Open MPI v4.0.0 (and beyond)?";
+
+$anchor[] = "mpi-1-deleted-functions";
+
+$a[] = "Starting with v4.0.0, Open MPI -- by default -- removes the
```

Comment:
So this is picky, but the en dash doesn't really fit here.  We should either use --- / an em dash, parenthetical, or a rewording.  Same with the identical text in the mpi-legacy section.